BUILD.gn

```diff
@@ -59,6 +59,15 @@ group("brave") {
       "//chrome"
     ]
   }
+
+  if (is_win && is_official_build) {
+    # Generate courgette binary by brave target instead of create_dist target.
+    # courgette.exe should be copied to the folder pointed by last_chrome_installer
+    # before running create_dist with build_delta_installer.
+    # When mini_installer target generates diff installer patch, it assumes
+    # that courgette.exe is already in last_chrome_installer folder.
+    deps += [ "//courgette:copy_courgette_binaries" ]
```

Comment:
I don't think this is the right place to put this. https://github.com/brave/brave-core/pull/5484/files#diff-769a945034c01d9bc9d33ecfa819c902R20 should be consolidated into a single dep on `//brave/installer/mini_installer` which has deps for `courgette:copy_courgette_binaries` conditionally on `build_delta_installer` and then on `//brave` instead of `//brave:packed_resources` and `//brave/build/win:copy_exe`

Reply:
also this list is incomplete and can be eliminated by just making the target dependent on `//brave` https://github.com/brave/brave-core/pull/5484/files#diff-769a945034c01d9bc9d33ecfa819c902R19